src/parser/keyword_helper.cpp

```diff
@@ -24,6 +24,13 @@ bool KeywordHelper::RequiresQuotes(const string &text) {
 	return IsKeyword(text);
 }
 
+string KeywordHelper::RemoveQuotes(const string &text) {
+	if (!RequiresQuotes(text)) {
+		return text;
+	}
+	return text.substr(1, text.length() - 2);
+}
+
```

Comment:
this is necessary in the very specific edge case in which users write SERIAL instead of serial - the transformer thinks it's an user type and adds quotes, and I want the process to be reversed

Reply:
I think it's perhaps better to make changes to one of the keyword lists (probably `unreserved`) in the grammar to let it recognize SERIAL as a native defined type, rather than fixing this confusion here.

Reply:
it is an unreserved keyword already (maybe I forgot to commit the file oopsies), else the parser would complain - but it is not a data type, so it thinks it's a user type

Reply:
Ah my mistake, then I understand the need